### PR TITLE
bugfix:containsPK ignoreCase

### LINKED ISSUE #1217: Rm cannot find primary key when primary key is Uppercase in sql ,using MySQL.
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
when I write a Insert SQL which the primary key is uppercase, RM cannot generate afterimge.
I find InsertExecutor.containsPK() judge if pk exists by comparing MySQL table metadata with the result of SQL recognizer. Which compare the lowercase PK and uppercase Pk.
When this table dosenot have auto-increase PK, RM throw exception.
### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

### Ⅵ. Environment:

- JDK version :
- OS :
- Others: Seata 0.6.1

### PR DESCRIPTION
**Pull request by slievrly**
Signed-off-by: slievrly <slievrly@163.com>

### Ⅰ. Describe what this PR did

bugfix:containsPK ignoreCase

### Ⅱ. Does this pull request fix one issue?
fix #1217 #1176

### CODE REVIEW

**Comment by VainEffort** [comment]
line 179.
Element needs to check null.

File: common/src/main/java/io/seata/common/util/CollectionUtils.java [hunk 1]
```diff
@@ -164,4 +166,19 @@ public static String encodeMap(Map<String, String> map) {
...
+     * To upper list list.
+     *
+     * @param sourceList the source list
+     * @return the list
+     */
+    public static List<String> toUpperList(List<String> sourceList) {
+        if (null == sourceList || sourceList.size() == 0) { return sourceList; }
+        List<String> destList = new ArrayList<>(sourceList.size());
+        for (String element : sourceList) {
+            destList.add(element.toUpperCase());
```
Comment (VainEffort) [inline common/src/main/java/io/seata/common/util/CollectionUtils.java]: Element needs to check null.

**Comment by slievrly** [comment]
> line 179.
> Element needs to check null.

great found.i will fix it.

File: common/src/main/java/io/seata/common/util/CollectionUtils.java (hunk 1 above)
Comment (slievrly) [reply common/src/main/java/io/seata/common/util/CollectionUtils.java]: fixed

**Pull request merged**